tests/test_smoke.py

```diff
@@ -3558,3 +3558,32 @@ def test_load_dump_yaml_config_equivalent(self):
                                                            exist_ok=True)
         for yaml_path in self._TEST_YAML_PATHS:
             self._check_equivalent(yaml_path)
+
+
+# ---------- Testing Multiple Resources ----------
+@pytest.mark.gcp
+def test_multiple_resources_ordered():
+    name = _get_cluster_name()
+    test = Test(
+        'multiple-resources-ordered',
+        [
+            f'sky launch -y -c {name} tests/test_yamls/test_multiple_resources_ordered.yaml | grep "is using user-specified accelerators list"',
+            f'sky logs {name} 1 --status',  # Ensure the job succeeded.
+        ],
+        f'sky down -y {name}',
+    )
+    run_one_test(test)
+
+
+@pytest.mark.gcp
+def test_multiple_resources_unordered():
+    name = _get_cluster_name()
+    test = Test(
+        'multiple-resources-unordered',
+        [
+            f'sky launch -y -c {name} tests/test_yamls/test_multiple_resources_unordered.yaml',
+            f'sky logs {name} 1 --status',  # Ensure the job succeeded.
+        ],
+        f'sky down -y {name}',
+    )
+    run_one_test(test)
```

Comment:
Do we want to add another test for the multiple resources?

Reply:
Sure!